vars/artifactsPipeline.groovy

```diff
@@ -89,6 +92,21 @@ def call(Map pipelineParams) {
                     }
                 }
             }
+            stage('Create SCT Runner') {
```

Comment:
This won't work unless you convert other steps to use longevity routines. Otherwise, runner will be just haning there and not going to be used

Reply:
Fixed. Thanks